### PR TITLE
feat(exchange): use fee from amm

### PR DESCRIPTION
**Pull request by Roznovjak**
Fixes: #262

### CODE REVIEW

**Review by enthusiastmartin — CHANGES_REQUESTED** [review]
Adjust these changes first because AMM does not have generic Fee type anymore.

File: pallets/xyk/src/lib.rs [hunk 1]
```diff
@@ -954,6 +954,11 @@ impl<T: Config> AMM<T::AccountId, AssetId, AssetPair, Balance> for Pallet<T> {
 	fn get_max_out_ratio() -> u128 {
 		T::MaxOutRatio::get()
 	}
+
+	fn get_fee(_pool_account_id: &T::AccountId) -> (u32, u32) {
+		let fee = T::GetExchangeFee::get();
```
Comment (vgantchev) [inline pallets/xyk/src/lib.rs:959]: so we are using `Fee::Default()` of 0,2% for any xyk pool? 
just double checking if this is the intention, bc in LBP it is customisable

**Review by vgantchev — APPROVED** [review]
lgtm

File: pallets/xyk/src/lib.rs (hunk 1 above)
Comment (Roznovjak) [reply pallets/xyk/src/lib.rs:959]: It's customisable in the config, but for all pools.

**Comment by Roznovjak** [comment]
closing this PR and opening new (#302) because this one is more complicated to merge

**Pull request closed without merging**